ambry-replication/src/main/java/com/github/ambry/replication/BackupCheckerThread.java

```diff
@@ -59,6 +72,22 @@ public class BackupCheckerThread extends ReplicaThread {
 
   private final Logger logger = LoggerFactory.getLogger(BackupCheckerThread.class);
   public static final String DR_Verifier_Keyword = "dr";
+  protected final AmbryCache fileDescriptorCache;
+  private final String basePath = "/export/content/lid/logs/ambry-server/i001";
+  public static final DateFormat dateFormat = new SimpleDateFormat("dd MMM yyyy HH:mm:ss:SSS");
+
+  protected class FileDescriptor implements AmbryCacheEntry {
+
+    SeekableByteChannel _seekableByteChannel;
+
+    public FileDescriptor(SeekableByteChannel seekableByteChannel) {
```

Comment:
Do we need random access to the file? 
I thought we only need append to the end of the file.

Reply:
For replicaCheckStatus, I reset the file pointer to start of the file since i don't want a history of how the token moves but just the current token and lag.